test/Spago/Install.purs

```diff
@@ -104,7 +104,7 @@ spec = Spec.around withTempDir do
     Spec.it "can't install (uncached) dependencies if offline" \{ spago, fixture } -> do
       spago [ "init" ] >>= shouldBeSuccess
       writeConfigWithEither
-      spago [ "install", "--offline", "either" ] >>= shouldBeFailureErr (fixture "offline.txt")
+      spago [ "install", "--offline", "identity" ] >>= shouldBeFailureErr (fixture "offline.txt")
```

Comment:
The trick to make this work is in the line above - `writeConfigWithEither` will write a `spago.yaml` that points `either` to a specific commit that we are certain is not in cache.
We could modify `writeConfigWithEither` (defined in line 237) to also point `identity` to its git repo at a specific commit, then this would work.

Once we do that we should duplicate this test and call it `"can install cached dependencies if offline"`, where we try to install `either`, which the previous test brought into the cache.

Reply:
Addressed in [5ac3611](https://github.com/purescript/spago/pull/1240/commits/5ac36114da694d230575fb86d993a9f7d21c1a2f)